### PR TITLE
Adding Disconnected Annotation to the CSV

### PR DESCRIPTION
**Pull request by orenc1**
so the operator would appear when filtering for Disconnected under Infrastructure Features in OperatorHub.

Signed-off-by: orenc1 <ocohen@redhat.com>

**Release note**:

```release-note
Adding Disconnected Annotation to the CSV
```

### CODE REVIEW

**Comment by ovirt-infra** [comment]
Hello contributor, thanks for submitting a PR for this project! 

I am the bot who triggers "standard-CI" builds for this project.
As a security measure, I will not run automated tests on PRs that are not from white-listed contributors.

In order to allow automated tests to run, please ask one of the project maintainers to review the code and then do one of the following:

1. Type `ci test please` on this PR to trigger automated tests for it.
2. Type `ci add to whitelist` on this PR to trigger automated tests for it and also add you to the contributor white-list so that your future PRs will be tested automatically. ( keep in mind this list might be overwritten if the job XML is refreshed, for permanent whitelisting, please follow #3 option )
3. If you are planning to contribute to more than one project, maybe it's better to ask them to add you to the project organization, so you'll be able to run tests for all the organization's projects.

**Comment by tiraboschi** [comment]
/hold
we are still not ready for this: before that we should be sure that we are using only sha digests everywhere and we are correctly populating relatedImages in the CSV

File: pkg/components/components.go
Comment (nunnatsa) [inline pkg/components/components.go:789]: I would get rid of the confusing `\"` by wrapping it with back quotes; something like:

```golang
"operators.openshift.io/infrastructure-features": `["Disconnected"]`,
```

Comment (orenc1) [reply pkg/components/components.go]: Thanks, done.

**Comment by orenc1** [comment]
> /hold
> we are still not ready for this: before that we should be sure that we are using only sha digests everywhere and we are correctly populating relatedImages in the CSV

Hi @tiraboschi, we are now using SHA digests in all related images in the CSV.
Can we add now that annotation?

**Comment by tiraboschi** [comment]
now .spec.relatedImages in the CSV is fully populated.
/unhold

**Review by tiraboschi — APPROVED** [review]

**Comment by nunnatsa** [comment]
This is not working:

/override ci/prow/hco-e2e-kv-smoke-gcp

**Comment by orenc1** [comment]
🎉

**Pull request merged**
